web/api/netdata-swagger.yaml

```diff
@@ -35,137 +34,219 @@ paths:
-      summary: 'List variables available to configure alarms for a chart'
-      description: 'Returns the basic information of a chart and all the variables that can be used in alarm and template health configurations for the particular chart or family'
+      summary: List variables available to configure alarms for a chart
+      description: >-
+        Returns the basic information of a chart and all the variables that can
+        be used in alarm and template health configurations for the particular
+        chart or family
       parameters:
         - name: chart
           in: query
-          description: 'The id of the chart as returned by the /charts call.'
+          description: The id of the chart as returned by the /charts call.
           required: true
           type: string
-          format: 'as returned by /charts'
+          format: as returned by /charts
       responses:
         '200':
-            description: 'A javascript object with information about the chart and the available variables'
-            schema:
-              $ref: '#/definitions/alarm_variables'
+          description: >-
+            A javascript object with information about the chart and the
+            available variables
+          schema:
+            $ref: '#/definitions/alarm_variables'
         '400':
-          description: 'Bad request - the body will include a message stating what is wrong.'
+          description: Bad request - the body will include a message stating what is wrong.
         '404':
-          description: 'No chart with the given id is found.'
+          description: No chart with the given id is found.
         '500':
-          description: 'Internal server error. This usually means the server is out of memory.'
+          description: >-
+            Internal server error. This usually means the server is out of
+            memory.
   /data:
     get:
-      summary: 'Get collected data for a specific chart'
-      description: |
-        The Data endpoint returns data stored in the round robin database of a chart.
+      summary: Get collected data for a specific chart
+      description: >
+        The Data endpoint returns data stored in the round robin database of a
+        chart.
       parameters:
         - name: chart
           in: query
-          description: 'The id of the chart as returned by the /charts call.'
+          description: The id of the chart as returned by the /charts call.
           required: true
           type: string
-          format: 'as returned by /charts'
+          format: as returned by /charts
           allowEmptyValue: false
           default: system.cpu
         - name: dimension
           in: query
-          description: 'zero, one or more dimension ids or names, as returned by the /chart call, separated with comma or pipe. Netdata simple patterns are supported.'
+          description: >-
+            zero, one or more dimension ids or names, as returned by the /chart
```

Comment:
Please, write `Z` in upper case in the word `zero`

Reply:
Fixed.